docs/src/main/sphinx/release/release-396.md

```diff
@@ -0,0 +1,51 @@
+# Release 396 (14 Sep 2022)
+
+## General
+
+* Improve performance of queries that process string data types. ({issue}`12798`)
+* Fix failure when querying a view which uses a table function.  ({issue}`13944`)
+
+## BigQuery connector
+
+* Add support for [column comments and table comments](/sql/comment). ({issue}`13882`)
+
+## ClickHouse connector
+
+* Allow pushdown of larger predicates by default. ({issue}`14029`)
```

Comment:
What does "larger predicates" mean? 

This should be rephrased in terms of "Improve performance when xxx"

Reply:
This is increasing the default domain compaction threshold so that we are less aggressive about simplifying predicates with large number of values into a range predicate. It benefits pushdown of dynamic filters in particular but is applicable to any predicate pushdown.
Maybe "Improve performance when large dynamic filters are collected in a join" ? Not sure if that is appropriate level of detail for release notes.